scout/commands/load/variants.py

```diff
@@ -125,55 +137,124 @@ def variants(
     if keep_actions:  # collect all variants with user actions for this case
         old_evaluated_variants = list(adapter.evaluated_variants(case_id, institute_id))
 
-    i = 0
-    for file_type in files:
-        variant_type = file_type["variant_type"]
-        category = file_type["category"]
+    def load_variant_files(
```

Comment:
Hopefully two functions will make it a little easier to read. They are similar, but not so much so that I think generalising would serve it yet. Three times is a charm?